### DISCUSSION TITLE
Adding disk drive feature for setup module type

### DISCUSSION THREAD

**Discussion by bbrk364** [netbox-community/devicetype-library#901] [Ideas]
The need of documentation of server w/ type of disk drive or as module type:
1. Storage type HDD: LFF 3.5", SFF 2.5" and 1.8"
2. storage type : SSF, 7.2K, 10K, 15K and etc
3. type of interface: SATA, SAS or uSATA, NVMe
4. storage size" 200GB, 480GB, 600GB, 500GB, 1.0TB, 2TB etc

**Comment by DanSheps** [comment]
This needs to be requested upstream in the NetBox repository first

**Comment by danner26** [comment]
As DanSheps said, this will need to be requested from the upstream first. In the meantime, if this answers your questons @bbrk364, can you please mark Dan's comment as the answer? Thank you
